theme/layouts/_navigation.ejs

```diff
@@ -1,9 +1,25 @@
-<nav>
-  <ul>
-    <% for (const {title, link} of site.navigation) {%>
-      <li>
-        <a href="<%- link %>" target="_blank"><%- title %></a>
-      </li>
-    <% } %>
-  </ul>
+<select id="mobile-nav">
+      <% for ({title, link, sub_items} of site.navigation) { %>
+        <option value="<%- title %>"><%- title %></option>
+      <% } %>
+</select>
+
+<nav id="main-navigation">
+  <% function recNav(navig) { %>
+    <ul> 
+      <% for ({title, link, sub_items} of navig) { %>
+
+        <li>
+            <a href="<%- link %>" target="_blank"><%- title %> </a>
```

Comment:
Seems like in the Drupal website links were not using `target="_blank"`, this will open links in the new tab.